nbproject/private/private.xml

```diff
@@ -2,6 +2,13 @@
 <project-private xmlns="http://www.netbeans.org/ns/project-private/1">
     <editor-bookmarks xmlns="http://www.netbeans.org/ns/editor-bookmarks/2" lastBookmarkId="0"/>
     <open-files xmlns="http://www.netbeans.org/ns/projectui-open-files/2">
-        <group/>
+        <group>
+            <file>file:/Users/batesjernigan/Desktop/code/Java/web_java/MyTwitter/src/java/business/User.java</file>
+            <file>file:/Users/batesjernigan/Desktop/code/Java/web_java/MyTwitter/web/login.jsp</file>
```

Comment:
ahh these might not work on your machine. I'm going to re push without this in the commit